util/memory/tracker.go

```diff
@@ -712,6 +712,28 @@ func (t *Tracker) setParent(parent *Tracker) {
 	t.parMu.parent = parent
 }
 
+// CountAllChildrenMemUse return memory used tree for the tracker
+func (t *Tracker) CountAllChildrenMemUse() map[string]int64 {
+	trackerMemUseMap := make(map[string]int64, 1024)
+	countChildMem(t, "", trackerMemUseMap)
+	return trackerMemUseMap
+}
+
+func countChildMem(t *Tracker, familyTreeName string, trackerMemUseMap map[string]int64) {
+	familyTreeName += "[" + strconv.Itoa(t.Label()) + "] <- "
+	trackerMemUseMap[familyTreeName] += t.BytesConsumed()
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	if len(t.mu.children) == 0 {
+		return
+	}
```

Comment:
Remove the check is also ok..
if `len(t.mu.children) == 0`,  `for _, sli := range t.mu.children` will skip.

Reply:
> Remove the check is also ok.. if `len(t.mu.children) == 0`, `for _, sli := range t.mu.children` will skip.

done.